app/src/main/java/com/example/benchproject/data/popular/movies/entity/MovieResponseRemote.kt

```diff
@@ -1,5 +1,18 @@
 package com.example.benchproject.data.popular.movies.entity
 
+import com.example.benchproject.data.popular.movies.repo.PopularMoviesDataRepository
+import com.example.benchproject.domain.popular.movies.entity.MovieModel
+
 data class MovieResponseRemote(
     val results: List<MovieRemote>
-)
+) {
+    fun toDomain() = this.results.map { movieRemote ->
+        MovieModel(
+            movieRemote.id,
+            movieRemote.name,
+            movieRemote.rating,
+            movieRemote.releaseDate,
+            PopularMoviesDataRepository.BASE_URL_FOR_IMAGE + movieRemote.imagePath
```

Comment:
This will be changed when PR #28 will be merged. Since `BASE_URL_FOR_IMAGE` will be in `MoviesConstants`

Reply:
And I changed it :D